src/presentation/components/TopBar/DesktopTopBar.tsx

```diff
@@ -39,7 +39,7 @@ export const DesktopTopBar = () => {
         ) : (
           <div className={styles.topBar__itemButton}>
             <Button onClick={() => setSignupModalIsOpen(true)}>{t('global.signup')}</Button>
-            <span className={styles.topBar__itemLink}>Or signin </span>
+            <span className={styles.topBar__itemLink}>Or sign in </span>
```

Comment:
Shouldnt define this message in your translations file as constant?

Reply:
Yes, you are right